src/main/java/uk/ac/ebi/eva/pipeline/io/mappers/VariantVcfFactory.java

```diff
@@ -344,7 +343,7 @@ protected void parseInfo(Variant variant, String fileId, String studyId, String
                     case "AF":
                         // TODO For now, only one alternate is supported
                         if (splits[1].equals("0")) {
-                            throw new NotAVariantException();
+                            throw new NotAVariantException("This can't be zero");
```

Comment:
This message is unclear, because when the exception is thrown the context will be lost. Something like `The value of the AF INFO field can't be zero` would be more descriptive. The same applies to AN.